app/index.js

```diff
@@ -19,6 +23,69 @@ ipcRenderer.on('ipcToRedux', (_, message) => {
   }
 });
 
+ipcRenderer.on('deeplink', (_, message) => {
+  handleDeeplink(message);
+});
+
+function handleDeeplink(message) {
+  const url = new URL(message);
+  const state = store.getState();
+  const isLocked = state.wallet.isLocked;
+  const params = url.searchParams;
+
+  analytics.track('deeplink', {
+    pathname: url.pathname,
+  });
+
+  let name, txt;
+  switch (url.pathname) {
+    case "//updaterecord":
+      name = params.get('name');
+      txt = params.get('txt');
+
+      if (txt) {
+        store.dispatch(setDeeplinkParams({ txt }));
+      }
+
+      if (isLocked) {
+        store.dispatch(setDeeplink(message));
+      } else {
+        history.push(`/domain_manager/${name}`);
+      }
+      return;
+    case "//openauction":
```

Comment:
Would it make more sense to have `openauction` be `openname`, to match more closely with the ecosystem's block explorers?   i.e. The on-chain auction/ownership page is the high level view of the name, so you're opening 
 (information for) the name.

Same line of thinking: `opendomain` could be `openmanager` or `namemanager` as name itself would be implied by a param.  I'm not sure if you'd prefer verb-prefixes, so maybe `openmanager` is best.

Reply:
updated